raft/storage_test.go

```diff
@@ -51,12 +51,8 @@ func TestStorageTerm(t *testing.T) {
 			}()
 
 			term, err := s.Term(tt.i)
-			if err != tt.werr {
-				t.Errorf("#%d: err = %v, want %v", i, err, tt.werr)
-			}
-			if term != tt.wterm {
-				t.Errorf("#%d: term = %d, want %d", i, term, tt.wterm)
-			}
+			require.Equal(t, tt.werr, err)
```

Comment:
maybe we should keep the index so that it is easier to find out which case failed, like

`require.Equal(t, tt.werr, err, "#%d", i)`